static/app/views/ai-analytics/index.tsx

```diff
@@ -0,0 +1,73 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import Feature from 'sentry/components/acl/feature';
+import {Alert} from 'sentry/components/alert';
+import * as Layout from 'sentry/components/layouts/thirds';
+import NoProjectMessage from 'sentry/components/noProjectMessage';
+import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Organization} from 'sentry/types';
+import withOrganization from 'sentry/utils/withOrganization';
+import AiAnalyticsFilters from 'sentry/views/ai-analytics/filters';
+
+type Props = {
+  children: React.ReactNode;
+  organization: Organization;
+};
+
+function AiAnalyticsContainer({organization, children}: Props) {
+  const [search, setSearch] = useState('');
+
+  function renderNoAccess() {
+    return (
+      <Layout.Page withPadding>
+        <Alert type="warning">{t("You don't have access to this feature")}</Alert>
+      </Layout.Page>
+    );
+  }
```

Comment:
Nitpit: this can be a separate component outside of this component.

Reply:
copypasta from elsewhere